src/modules/endpoint/sagas/loadCities.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import type { Saga } from 'redux-saga'
+import { createCitiesEndpoint, Payload } from '@integreat-app/integreat-api-client'
+import { call, put } from 'redux-saga/effects'
+import request from '../request'
+import { baseUrl } from '../constants'
+import type { DataContainer } from '../DataContainer'
+import CityModel from '@integreat-app/integreat-api-client/models/CityModel'
+import type { PushCitiesActionType } from '../../app/StoreActionType'
+
+function * loadCities (
+  dataContainer: DataContainer,
+  forceRefresh: boolean
+): Saga<Array<CityModel>> {
+  let cities: Array<CityModel>
+  if (!forceRefresh && (yield call(() => dataContainer.citiesAvailable()))) {
```

Comment:
```suggestion
  const citiesAvailable = yield call(() => dataContainer.citiesAvailable())
  if (!forceRefresh && citiesAvailable) {
```

Reply:
Hmm... if forceRefresh is true this might cause an unnecessary fs call, therefore I'd like to leave it like this as a mini optimzation vs. a mini code readability improvement :sweat_smile: